_includes/feedback.html

```diff
@@ -0,0 +1,14 @@
+<div class="whats-new-post" style="padding: 15px; background: #e4f0f9; border-left: 2px solid blue;">
+
+<h4> Give us your feedback! </h4>
+
+<p>
+
+Every quarter OpenSearch obtains feedback from users. This helps us make improvements to the OpenSearch Project. Please take a few minutes to take this survey.
+
+</p>
+
+<a href="https://amazonmr.au1.qualtrics.com/jfe/form/SV_1BxJNrtCo4LbweW">Take the survey.</a>
```

Comment:
Given this section wont be changing (aside from the href's why dont we put it above the "Whats New" Title. That way it is clear this is not a blog but a static section.

Reply:
Agree - this looks fantastic - just move it above the 'What's New' title and we're good to go @nateynateynate